scripts/gh_matrix_builder.py

```diff
@@ -58,7 +58,7 @@ def build_release_config(overrides):
     "name": "Release",
     "build_type": "Release",
     "pg_build_args": "",
-    "tsdb_build_args": "-DWARNINGS_AS_ERRORS=OFF",
+    "tsdb_build_args": "",
```

Comment:
I think it would be better to a) let the cron jobs run the Release builds (with defaults) and b)  not disable warnings with errors for our regular github CI . Otherwise we would be running PRs without enabling this option.

Reply:
As we discussed, it will also capture any new warnings generated and make sure they are not left unattended. The Coverity build and the Apache build still use the default here, so we check this as well.